src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -322,4 +320,5 @@
           authorization-fn
           whitelist-path
           (get-in ca-settings [:access-control :certificate-status]))
+        (i18n/locale-negotiator)
```

Comment:
ok.  well, he's wrong.  :)  But since you're on his team for the moment you can leave it like this.